src/test/java/lambda/part2/exercise/Exercise1.java

```diff
@@ -29,35 +30,29 @@ public void sameAgesCheckerUsingBiPredicate() {
         Person person2 = new Person("Дмитрий", "Гущин", 33);
         Person person3 = new Person("Илья", "Жирков", 22);
 
-        // TODO создать переменную sameAgesChecker: (Person, Person) -> boolean, используя BiPredicate
-
-        // assertTrue(sameAgesChecker.test(person1, person2));
-        // assertFalse(sameAgesChecker.test(person1, person3));
-        // assertFalse(sameAgesChecker.test(person2, person3));
+        BiPredicate<Person, Person> sameAgesChecker = (person, other) -> Objects.equals(person.getAge(), other.getAge());
 
-        // FIXME удалить при реализации
-        throw new UnsupportedOperationException("Not implemented");
+        assertTrue(sameAgesChecker.test(person1, person2));
+        assertFalse(sameAgesChecker.test(person1, person3));
+        assertFalse(sameAgesChecker.test(person2, person3));
     }
 
-    // TODO метод getFullName: Person -> String, извлекающий из объекта Person строку в формате "имя фамилия".
-    // private static ... getFullName(...) {
+    private static String getFullName(Person person) {
+        return person.getFirstName() + " " + person.getLastName();
```

Comment:
We have getFullName method at Person class. You can use it instead of concatenation.